.env.appStore.example

```diff
@@ -125,4 +125,11 @@ SALESFORCE_CONSUMER_SECRET=""
 ZOHOCRM_CLIENT_ID=""
 ZOHOCRM_CLIENT_SECRET=""
 
+# ALBY
+# Used for the Alby payment app / receiving Alby payments
+# Get it from: https://getalby.com/developer/oauth_clients
+# Set callbackUrl to /api/integrations/alby/alby-webhooks
```

Comment:
it should be `webhook` not `webhooks` - I will update in a follow up commit. 

Thanks for updating the comment